### PR TITLE
Minor fixes for preprocessing smooth uv spectrum

### PR DESCRIPTION
**Pull request by jychoi-hpc**
Minor fixes for preprocessing smooth uv spectrum on Frontier.

### CODE REVIEW

File: examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py [hunk 1]
```diff
@@ -151,9 +151,8 @@ def __init__(self, dirpath, dftb_node_types, var_config, dist=False, sampling=No
                 extract_path = extract_tar_file(fullpath, tmpfs)
                 mdirs = get_mol_dir_list(extract_path)
                 assert len(mdirs) > 0, f"No molecules found in tar file {fullpath} extracted at {extract_path}"
-                for mdir in mdirs:
+                for mdir in iterate_tqdm(mdirs, verbosity_level=2, desc="Processing"):
```
Comment (kshitij-v-mehta) [inline examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py:154]: Is line 155 redundant? `data_object` is being set on line 156 as well.

**Review by kshitij-v-mehta — COMMENTED** [review]
Also, I think [line 165](https://github.com/kshitij-v-mehta/HydraGNN/pull/4/files#diff-db9da2192ad7199e7eed65fe2d3e19bbe0ab468a0c6575b8a494c177a0a926ebR165)

should have `fullpath` instead of `mdir` as the last function argument. 
So the line should be 
`data_object = self.transform_input_to_data_object_base(dirpath, fullpath)`

I will make that change and update this branch.

File: examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py [hunk 2]
```diff
@@ -190,8 +189,9 @@ def __transform_DFTB_UV_input_to_data_object_base(self, raw_data_path, dir):
         data_object = None
 
         # collect information about molecular structure and chemical composition
+        #print ("__transform_DFTB_UV_input_to_data_object_base:", raw_data_path, dir)
         try:
-            pdb_filename = raw_data_path + '/' + dir + '/' + 'smiles.pdb'
+            pdb_filename = dir + '/' + 'smiles.pdb'
```
Comment (kshitij-v-mehta) [inline examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py:192]: Should we just remove `raw_data_path` from the function signature?

File: examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py (hunk 1 above)
Comment (jychoi-hpc) [reply examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py:154]: @kshitij-v-mehta you are right. It is duplicated and that is why I am getting errors with atomicdescriptors. I think using `transform_input_to_data_object_base` is the right way. I will fix.

File: examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py (hunk 2 above)
Comment (jychoi-hpc) [reply examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py:192]: You are right.  `raw_data_path` is not needed anymore. I will remove.

File: examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py (hunk 1 above)
Comment (jychoi-hpc) [reply examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py:154]: I found `dftb_to_graph` is the correct way. I switched.

**Pull request merged**